addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -364,7 +364,11 @@ export class ProductScreen extends Component {
             if (filteredList.length >= 100) {
                 break;
             }
-            if (!excludedProductIds.includes(product.id) && product.available_in_pos) {
+            if (
+                !excludedProductIds.includes(product.id) &&
+                product.available_in_pos &&
+                product.canBeDisplayed
```

Comment:
I'm not sure if we need to create a getter for that, maybe directly use product.active here. Is easier to understand. 

Or you can remove available in pos from here and use it directly in your getter